### PR TITLE
Node-RED Pillar Page Updates

### LINKED ISSUE #376: Create a page of Node-RED hardware partners
### Story

_No response_

### Description

Need a page that links to all the hardware vendors that use Node-RED.

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- New Hero Section
- New 6 point highlights for Node-RED
- Add Hardware Partners

https://github.com/flowforge/website/assets/99246719/f2d39e5e-ef18-4e5c-bc67-a525f0ad50c7.mp4

### Outstanding Items:

- [x] Full list of NR hardware partners
- [x] Caveat/note on hardware list that they have not endorsed FF
- [x] Hero b/g

## Related Issue(s)

First pass of updates for #739 
Closes #376 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes

### CODE REVIEW

**Comment by joepavitt** [comment]
updates to the top 6 points and associated icons:

Screenshot 2023-05-19 at 11 50 45

**Comment by joepavitt** [comment]
> I would remove the hardware partner section. I don't think it fits with the flow of the content or the context of the page.

Understand. Where would you envision us linking to this hardware page? In the interest of bias for action, may I propose we push what we have, as it is okay, and then iterate to remove and separate to it's own page later. Was expecting this to be merged whilst i was away.

**Comment by iskerrett** [comment]
> > The list of hardware partners is also not complete.
> 
> Will add "BLIIOT", had missed them off as it required me to create a custom logo, time consuming given the benefit of including them and blocking this page update. Had also added "Multitech" which was missing from your [original list](https://github.com/flowforge/website/issues/376#issuecomment-1414287380)

Unfortunately that list is way out of date. It is from February. Here is the most recent list.  https://flowforge.com/blog/2023/03/integration-platform-for-edge-computing/

Sorry, I wasn't aware you were doing a hardware page.

**Comment by iskerrett** [comment]
> > I would remove the hardware partner section. I don't think it fits with the flow of the content or the context of the page.
> 
> Understand. Where would you envision us linking to this hardware page?
> 
> In the interest of bias for action, may I propose we push what we have, as it is okay, and then iterate to remove and separate to it's own page later. Was expecting this to be merged whilst i was away.

If the purpose of this redesign is to improve readability then I think we should remove it. 

I wasn't actually involve in the redesign so I am not sure where we would link a hardware page. One idea is we change some text to link to hardware providers.  I can certainly make some suggestions. However, I don't think the hardware page is a priority right now.

**Comment by joepavitt** [comment]
Understood - so we'll remove that section for now and move forward without it.

**Comment by joepavitt** [comment]
@iskerrett fyi - Hardware Partners removed

**Comment by Yndira-FlowForge** [comment]
@joepavitt I'm not seeing the background of the hero section, did you remove it? (I'm seeing the netlify preview)

**Comment by joepavitt** [comment]
Well spotted @Yndira-FlowForge not sure what's happened there - on something else at the moment, will take a look later.

File: src/css/style.nohero.css [hunk 1]
```diff
@@ -14,8 +14,8 @@
 }
 
 .nohero h2 {
```
Comment (ZJvandeWeg) [inline src/css/style.nohero.css:16]: ✂️

File: src/images/hardware-partners/advantech.svg
Comment (ZJvandeWeg) [inline src/images/hardware-partners/advantech.svg:1]: Can we remove the logo's for now?

**Comment by ZJvandeWeg** [comment]
This iteration as is seems a clear step forward, as such I'm merging. Tomorrow is another day for further improvements.

**Pull request merged**